corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -98,10 +98,14 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
             }
         });
 
+        self.display = ko.observable(typeof params.display !== 'undefined' ? params.display : "");
```

Comment:
Should this be `display` or header? Would be good to be consistent as to what it's called. Personally I think `display` is better since for sort only fields there is no `header`.